app/routes/routes_helper_function.py

```diff
@@ -0,0 +1,14 @@
+from flask import abort, make_response
```

Comment:
Beautiful file all around; this workhorse of a helper function really DRY'd out the routes files. Since we might add more helper functions, our file could be named a more generic, `routes_helpers.py`.